### PR TITLE
feat/ambient occlusion

### PR DESCRIPTION
**Pull request by blanxii**
Add default ambient occlusion properties when provided, the threshold it's really low check for the default example

image

this is the result with ambient occlusion applied

image

Normal one:

image

### CODE REVIEW

File: src/components/Avatar/Avatar.component.tsx [hunk 1]
```diff
@@ -262,14 +277,27 @@ const Avatar: FC<AvatarProps> = ({
...
-          intensity={bloom?.intensity}
-          kernelSize={bloom?.kernelSize}
-          mipmapBlur={bloom?.mipmapBlur}
+            luminanceThreshold={bloom?.luminanceThreshold}
+            luminanceSmoothing={bloom?.luminanceSmoothing}
+            intensity={bloom?.intensity}
+            kernelSize={bloom?.kernelSize}
+            mipmapBlur={bloom?.mipmapBlur}
         />
+        <>
```
Comment (Zaehiel) [inline src/components/Avatar/Avatar.component.tsx:288]: Fragment not necessary

File: src/components/Avatar/Avatar.component.tsx [hunk 2]
```diff
@@ -262,14 +277,27 @@ const Avatar: FC<AvatarProps> = ({
...
-          luminanceThreshold={bloom?.luminanceThreshold}
-          luminanceSmoothing={bloom?.luminanceSmoothing}
-          intensity={bloom?.intensity}
-          kernelSize={bloom?.kernelSize}
-          mipmapBlur={bloom?.mipmapBlur}
+            luminanceThreshold={bloom?.luminanceThreshold}
+            luminanceSmoothing={bloom?.luminanceSmoothing}
+            intensity={bloom?.intensity}
+            kernelSize={bloom?.kernelSize}
+            mipmapBlur={bloom?.mipmapBlur}
```
Comment (Zaehiel) [inline src/components/Avatar/Avatar.component.tsx:286]: What's causing this auto-reformatting?

Comment (blanxii) [reply src/components/Avatar/Avatar.component.tsx:286]: reformat code, for some reason, we miss the .editorconfig where we set those standards

File: src/components/Avatar/Avatar.component.tsx (hunk 1 above)
Comment (Zaehiel) [reply src/components/Avatar/Avatar.component.tsx:288]: Yes I get it, is fine!

**Review by Zaehiel — APPROVED** [review]

**Pull request merged**